util/tracing/trace_to_csv.c

```diff
@@ -116,156 +118,161 @@ size_t read_and_write_trace() {
         if (trigger_name == NULL) {
             trigger_name = "NO TRIGGER";
         }
-        fprintf(output_file, "%s, %s, %d, %d, %lld, %d, %lld, %s, %lld\n",
-                trace_event_names[trace[i].event_type],
-                reactor_name,
-                trace[i].src_id,
-                trace[i].dst_id,
-                trace[i].logical_time - start_time,
-                trace[i].microstep,
-                trace[i].physical_time - start_time,
-                trigger_name,
-                trace[i].extra_delay
-        );
-        // Update summary statistics.
-        if (trace[i].physical_time > latest_time) {
-            latest_time = trace[i].physical_time;
-        }
-        if (object_instance >= 0 && summary_stats[NUM_EVENT_TYPES + object_instance] == NULL) {
-            summary_stats[NUM_EVENT_TYPES + object_instance] = (summary_stats_t*)calloc(1, sizeof(summary_stats_t));
-        }
-        if (trigger_instance >= 0 && summary_stats[NUM_EVENT_TYPES + trigger_instance] == NULL) {
-            summary_stats[NUM_EVENT_TYPES + trigger_instance] = (summary_stats_t*)calloc(1, sizeof(summary_stats_t));
-        }
+        if ((trace[i].logical_time - start_time) >= trace_start_time
+        && (trace[i].logical_time - start_time) < trace_end_time) {
```

Comment:
Nit: I think the correct indentation here should be like this:
```
        if ((trace[i].logical_time - start_time) >= trace_start_time
            && (trace[i].logical_time - start_time) < trace_end_time) {
```

Reply:
Resolved in [14ecd1c](https://github.com/lf-lang/reactor-c/pull/320/commits/14ecd1c2e76936257403806efe3e378f347d189a).